spec/support/helpers.rb

```diff
@@ -0,0 +1,26 @@
+module Helpers
+
+  def sign_up(user)
+    visit ('/')
+    click_button 'Sign up'
+    fill_in 'email',    with: user.email
+    fill_in 'name',     with: user.name
+    fill_in 'username', with: user.username
+    fill_in 'password', with: user.password
+    fill_in 'password_confirmation', with: user.password_confirmation
+    click_button 'Register'
+  end
+
+  def log_in(user)
+    visit ('/')
+    click_button 'Log in'
+    fill_in 'username', with: user.username
+    fill_in 'password', with: user.password
+    click_button 'Enter'
+  end
+
+  def sign_out(user)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `sign_out(*)` if you want the method to accept any arguments but don't care about them.